CONTRIBUTING.adoc

```diff
@@ -1,13 +1,47 @@
-= Code formatting guidelines
+= Contributing to Spring Cloud GCP
+
+We graciously welcome contributions to Spring Cloud GCP.
+
+== Building the Project
+
+The first step to contributing is to fork the repository and clone it onto your machine.
+
+1. Ensure you have Java version 8 or later installed on your machine.
+
+1. Install the https://cloud.google.com/sdk/docs/[Google Cloud SDK].
```

Comment:
This should apply only if you want to run integration tests and samples.
It should be possible to run `mvn clean install` without gcloud.
I believe that's how it's built on Jenkins currently.

Reply:
Done; I reorganized the info to express this better.